From 4948c62408c56c1b5e3cfed9a628aa42cbbb54ee Mon Sep 17 00:00:00 2001
From: Renat Khalikov <renat_khalikov@trimble.com>
Date: Wed, 19 Apr 2023 20:23:31 -0600
Subject: [PATCH] Include misc-modules and scull

---
 Makefile | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 7b08c49..c4f2e2e 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,5 @@

-SUBDIRS =  misc-progs misc-modules \
-           skull scull scullc scullp lddbus sculld scullv shortprint simple tty \
-	   pci usb\
-	   sbull snull short
+SUBDIRS =  misc-modules scull

 all: subdirs
